README.md

```diff
@@ -353,6 +353,7 @@ Add your plugin to [the `plugins.json` file](https://github.com/serverless/commu
 | **[Webpack - `serverless-plugin-webpack`](https://github.com/goldwasserexchange/serverless-plugin-webpack)** <br/> by [goldwasserexchange](http://github.com/goldwasserexchange) <br/> A serverless plugin to automatically bundle your functions individually with webpack | ![Github Stars](https://img.shields.io/github/stars/goldwasserexchange/serverless-plugin-webpack.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-plugin-webpack.svg?label=Downloads&style=for-the-badge)|
 | **[Webpack - `serverless-webpack`](https://github.com/serverless-heaven/serverless-webpack)** <br/> by [serverless-heaven](http://github.com/serverless-heaven) <br/> Serverless plugin to bundle your lambdas with Webpack | ![Github Stars](https://img.shields.io/github/stars/serverless-heaven/serverless-webpack.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-webpack.svg?label=Downloads&style=for-the-badge)|
 | **[Website Domain - `serverless-website-domain`](https://github.com/williamsandonz/serverless-website-domain)** <br/> by [williamsandonz](http://github.com/williamsandonz) <br/> A plugin that creates Route 53 records that point to your Cloudfront hosted static website, including www/non-www redirects. | ![Github Stars](https://img.shields.io/github/stars/williamsandonz/serverless-website-domain.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-website-domain.svg?label=Downloads&style=for-the-badge)|
+| **[Output to ENV - `serverless-output-to-env`](https://github.com/williamsandonz/serverless-output-to-env)** <br/> by [williamsandonz](http://github.com/williamsandonz) <br/> A plugin that creates Route 53 records that point to your Cloudfront hosted static website, including www/non-www redirects. | ![Github Stars](https://img.shields.io/github/stars/williamsandonz/serverless-output-to-env.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-output-to-env.svg?label=Downloads&style=for-the-badge)|
```

Comment:
The description here does not look correct - could you please look into it? Thanks in advance :bow: